tsconfig.base.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compileOnSave": false,
+  "compilerOptions": {
+    "baseUrl": "./",
+    "outDir": "./dist/out-tsc",
+    "sourceMap": true,
+    "declaration": false,
+    "downlevelIteration": true,
+    "experimentalDecorators": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "importHelpers": true,
+    "target": "es2015",
+    "typeRoots": [
```

Comment:
typeRoots are removed so maybe we don't need this here as well.
See the defaults in the TSConfig Reference:
- https://www.typescriptlang.org/docs/handbook/compiler-options.html#compiler-options
- https://www.typescriptlang.org/tsconfig#types